R/incidence.R

```diff
@@ -155,6 +155,9 @@ incidence <- function(dates, interval = 1L, ...) {
 #' @export
 #' @rdname incidence
 incidence.default <- function(dates, interval = 1L, ...) {
+  if (is.character(dates)) {
+    stop('Input is a character. Did you forget to convert to Date?')
+  }
   check_dates(dates)
```

Comment:
I would remove the `check_dates(dates)` and replace by a generic error:
```
msg <- 'Unknown date input; accepted formats are...'
stop(msg)
```
kinda stuff